i18n/es.json

```diff
@@ -515,6 +515,7 @@
   "admin.field_names.maxChannelsPerTeam": "Máximo número de Canales por Equipo",
   "admin.field_names.maxNotificationsPerChannel": "Cantidad máxima de Notificaciones por Canal",
   "admin.field_names.maxUsersPerTeam": "Máximo de usuarios por equipo",
+  "admin.field_names.minimumUsernameLength": "Longitud mínima del nombre de usuario",
```

Comment:
We only include the `en.json` string, all the other strings are managed through our translations server (https://translate.mattermost.com/).

Reply:
I was getting failures in this test https://github.com/mattermost/mattermost-webapp/blob/master/utils/admin_console_index.test.jsx that only cleared after I added the Spanish translation.

Reply:
Yes, that makes sense, because the test is doing the check against the translations, I didn't think about this exact case. Not a super cool solution, but we could change the search text used for the tests to not involve this change.